docs/get-started/store-and-retrieve/set-up.md

```diff
@@ -39,233 +39,7 @@ A lite-node lets your computer interact with the Filecoin network without having
-    ```
-    :::
-
-1. Check if you have [Homebrew](https://brew.sh/) installed:
-
-    ```shell
-    brew --version 
-
-    > Homebrew 3.0.11
-    ```
-
-    Any version of Homebrew will work, but version 3.0.0 and above is preferred.
-
-    :::warning
-    If the above command returns an error, you likely don't have Homebrew installed. Run the following command to install Homebrew:
-
-    ```shell
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-    ```
-
-    If you run into errors installing Homebrew, there may be something wrong with your macOS installation. Check out the [Homebrew documentation](https://brew.sh/) for more information on how you can it installed.
-    :::
-
-1. Use Homebrew to install the following packages:
-
-    ```shell
-    brew install go bzr jq pkg-config rustup hwloc
-    ```
-
-Next up, you'll install a Lotus lite-node on your computer.
-
-#### Install Lotus
-
-The installation steps are different depending on which CPU your Mac is using:
-
-- [AMD and Intel-based Macs](#amd-and-intel-based-macs)
-- [M1-based Macs](#m1-based-macs)
-
-If you are unsure which CPU your Mac is using, check out the [troubleshooting steps](./troubleshooting/#which-cpu-does-my-mac-have).
-
-##### AMD and Intel-based Macs
-
-1. Download the Lotus repository from GitHub:
-
-    ```shell
-    git clone https://github.com/filecoin-project/lotus.git
-    cd lotus
-    ```
-
-1. Build the `lotus` binary, but don't run anything just yet:
-
-    ```shell
-    make clean && make all
-    sudo make install
-    ```
-
-    If you run into errors here, it may be because your computer did not install some dependencies properly. Check out the [Troubleshooting](troubleshooting) section, or see the [Lotus GitHub repository](https://github.com/filecoin-project/lotus) for more help.
-
-1. You should now have the `lotus` executable ready to run on your computer. 
-
-    ```shell
-    lotus --version
-
-    > lotus version 1.7.0
-    ```
-
-[Head over to the next section to run your Lotus lite-node ↓](#run-a-lotus-lite-node)
-
-##### M1-based Macs
-
-Because of the novel architecture of the M1-based Mac computers, you must set some specific environment variables before creating the `lotus` executable.
-
-1. Clone the [Lotus repository](https://github.com/filecoin-project/lotus) from GitHub: 
-
-    ```shell   
-    git clone https://github.com/filecoin-project/lotus
-    cd lotus
-    ```
-
-1. Pull-in the submodules:
-
-    ```shell
-    git submodule update --init --recursive
-    ```
-
-1. Create necessary environment variable to allow Lotus to run on ARM architecture:
-
-    ```shell
-    export GOARCH=arm64
-    export CGO_ENABLED=1
-    export LIBRARY_PATH=/opt/homebrew/lib
-    ```
-
-1. Move into the `extern/filecoin-ffi` directory and checkout to the `m1-portable` branch:
-
-    ```shell
-    cd extern/filecoin-ffi
-    git fetch -a
-    git checkout m1-portable
```

Comment:
This instruction is out of date in addition to being redundant. later versions of the filecoin-ffi have m1 support and the m1-portable branch has been deleted.